cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/store/remote/RemoteStoreClient.java

```diff
@@ -45,6 +55,12 @@
  */
 public class RemoteStoreClient {
```

Comment:
Perhaps rename to make this more generic, since it seems to be used by all 'system/CDAP platform' ops in programs?